from_mwt_ds/DataScience/vw_executor/tests/test_Vw.py

```diff
@@ -0,0 +1,426 @@
+import unittest
+from vw_executor.vw import *
+from vw_executor.vw_opts import Grid
+import pandas as pd
+from pathlib import Path
+import shutil
+
+def reset_cache_folder(path):
+    p = Path(path)
+    if p.exists() and p.is_dir():
+        shutil.rmtree(p)
+
+class TestVw(unittest.TestCase):
+    input1 = 'vw_executor/tests/data/cb_0.json'
+    input2 = 'vw_executor/tests/data/cb_1.json'
+    input_ccb = 'vw_executor/tests/data/ccb_0.json'
+
+    def test_data_shape(self):
```

Comment:
nit: I usually prefer to split up different test cases into named tests, so that when one fails I quickly know which one, but that's a style thing so up to you

Reply:
good point, let me do that